package.json

```diff
@@ -6,14 +6,17 @@
     "@testing-library/jest-dom": "^5.11.10",
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.8.3",
+    "@types/actioncable": "^5.2.3",
     "@types/jest": "^26.0.22",
     "@types/node": "^14.14.37",
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.3",
+    "actioncable": "^5.2.5",
     "chess.js": "^0.11.0",
     "chessboardjsx": "^2.4.7",
     "cypress": "^7.0.0",
     "react": "^17.0.2",
+    "react-actioncable-provider": "^2.0.0",
```

Comment:
Turns out the most popular actioncable package for React ... doesn't actually work.